src/bot/mtproto-api/workrooms.ts

```diff
@@ -0,0 +1,197 @@
+import { Context, SupportedEvents } from "#root/types/context";
+import { CallbackResult } from "#root/types/proxy.js";
+import { MtProto } from "./bot/mtproto";
+import { Api } from "telegram";
+import { addCommentToIssue } from "#root/helpers/add-comment-to-issues.js";
+
+function isPriceLabelChange(label: string): boolean {
+  return label.toLowerCase().includes("price");
+}
```

Comment:
Why price label? What is this used for? I'm not really understanding the logic below.

Reply:
You said that a workroom should only be created after the task is priced, is this no longer the case?

Reply:
That makes sense I just couldn't understand your logic. I recall seeing a `!` operator in the diff, which means when a label is set but not price label. 

Anyways, the workroom should only be created once its a project in progress. I guess a minor optimization/adjustment (lets see after actually trying this in production though) would be to generate the workroom when somebody is working on it, like when the project is officially started. Otherwise the conversation should be right on GitHub for DAO contributor convenience to peruse.